github/client.go

```diff
@@ -152,6 +161,127 @@ func (c *Client) ReadReleaseByTag(ctx context.Context, tkn string, repoURL url.U
 	return model.ToSvcGithubRelease(rls)
 }
 
+// TODO once svcmodel.Release has git tag field, separate gitTag parameter wont be needed
+func (c *Client) UpsertRelease(ctx context.Context, tkn string, repo svcmodel.GithubRepo, rls svcmodel.Release, gitTag string) error {
```

Comment:
Once #116 is merged, I dont need to pass gitTag, because gitTag will be part of Release struct.